<!--This is an example configuration file from our use at Max Planck Institute for Intelligent Systems. We want to subscribe the ROS image topics "overlay_output_left" and "overlay_output_right" using the keying function, to overlay AR content on the input images acquired by a da Vinci stereo endoscopes. We use a DeckLink Quad 2.-->

<launch>     

     <node pkg="hi_decklink_ros" type="subscriber" name="camera_left"  output="screen">
	 
     	<param name="decklink_device" type="str" value="DeckLink Quad (4)" />
		<!--name of interface onto which the images will be written-->
	<param name="image_format" type="str" value="HD1080i5994" />
	<!-- ROS topic from which to read the images-->
	<param name="topic" type="str" value="overlay_output_left" />
	<!-- enable keying on the card - images can be overlayed-->
	<param name="keying" type="bool" value="true" />
	<!--opacity of the keyed images-->
	<param name="opacity" type="int" value="150" />
     </node>

     <node pkg="hi_decklink_ros" type="subscriber" name="camera_right"  output="screen">
     	<param name="decklink_device" type="str" value="DeckLink Quad (3)" />
	<param name="image_format" type="str" value="HD1080i5994" />
	<param name="topic" type="str" value="overlay_output_right" />
	<param name="keying" type="bool" value="true" />
	<param name="opacity" type="int" value="150" />
     </node>

</launch>
